### ISSUE TITLE
[Story] Clarify the use of `redoc-cli`, `redocly` and `openapi`

### ISSUE THREAD

**Issue by tschaffter** [Sage-Bionetworks/sage-monorepo#1779]
### Description

- The project `openchallenges-api-docs` currently uses `redoc-cli`, but soon will use `redocly` binary to build API docs.
- The project `openchallenges-api-description` uses `openapi` to build and preview docs, which actually duplicates the project `openchallenges-api-docs`.

### Have you linked this story to a GitHub Project?

- [X] I have linked this story to a GitHub Project and set its metadata.

**Comment by tschaffter** [comment]
## Binaries

```console
$ ls -an node_modules/.bin/openapi
lrwxrwxrwx 1 1000 1001 26 May 17 02:33 node_modules/.bin/openapi -> ../@redocly/cli/bin/cli.js

$ ls -an node_modules/.bin/redoc-cli
lrwxrwxrwx 1 1000 1001 21 May 17 02:33 node_modules/.bin/redoc-cli -> ../redoc-cli/index.js

$ ls -an node_modules/.bin/redocly 
lrwxrwxrwx 1 1000 1001 26 May 17 02:33 node_modules/.bin/redocly -> ../@redocly/cli/bin/cli.js
```

The binaries `openapi` and `redocly` are actually the same binaries, which is great because it's the two that we use.

From https://www.npmjs.com/package/redoc-cli:

> DEPRECATED: this package is deprecated. Use npx @redocly/cli build-docs <api> instead.

**Comment by tschaffter** [comment]
## Conclusion

Implemented in #1776 

- Remove the packages `redoc` and `redoc-cli`
- Update the project `openchallenges-api-description` to use the binary `redocly` instead of `openapi`
